packages/create-amplify/README.md

```diff
@@ -19,3 +19,19 @@ In a frontend project folder or empty folder, run `npm create amplify`.
    a. Do not install or use `amplify` globally
    b. Use the same package manager for all commands in a given project
    c. When running E2E tests locally without a package manager, set `npm_config_user_agent` to `npm`.
+
+# Manual Alternative
```

Comment:
This table is great for comparison of different package managers and finding their differences, maybe not so much as steps that customers can follow for their package manager.

I think this belongs in the docs site with a section per package manager to list commands/steps to run for scaffolding an amplify project. what do you think @josefaidt?

Reply:
is this section necessary for the readme? we can have a "manual install" page in the docs but this is very granular and we only document npm flows in the docs site

Reply:
In that case we can remove this section from the PR

Reply:
Removed 21a56b2947